synnax/pkg/api/hardware.go

```diff
@@ -275,6 +340,17 @@ func (svc *HardwareService) RetrieveDevice(ctx context.Context, req HardwareRetr
 	if hasMakes {
 		q = q.WhereMakes(req.Makes...)
 	}
+	err := q.Entries(&res.Devices).Exec(ctx, nil)
```

Comment:
@LeonLiur You're executing the query twice here, which means  Please fix this ASAP as its blocking release. Also, check for other places where you are doing this.